src/main/java/core/selenium/WebElementAction.java

```diff
@@ -15,11 +15,14 @@
 import org.openqa.selenium.interactions.Action;
 import org.openqa.selenium.interactions.Actions;
 import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
 
 /**
  * This class the common web element actions.
  */
 public class WebElementAction {
+    private WebDriver driver;
```

Comment:
This instance is unnecesary since we have a Singleton to use de driver and the wait

Reply:
Ok It was solved